tests/test_masking.py

```diff
@@ -26,6 +27,27 @@ def setUp(self):
             self.imlist.append(im)
         self.stack = ImageStack(self.imlist)
 
+    def test_apply_trajectory_mask(self):
+        starting_pixel = (20, 20)
+        velocity = (50.0, -10.0)
+        radius = 5
+        trj = make_trajectory(starting_pixel[0], starting_pixel[1], velocity[0], velocity[1])
+        bitmask = 1
+
+        masked_stack = mask_trajectory(trj, self.stack, radius, bitmask)
```

Comment:
You also need to remove the bitmask argument here (and can delete the definition on line 35).

Reply:
Ah yes, my bad. Removed both